src/apps/material/material.tsx

```diff
@@ -156,8 +159,14 @@ const Material: React.FC<MaterialProps> = ({ wid }) => {
       )}
       {currentManifestation && (
         <>
-          <FindOnShelfModal manifestation={currentManifestation} />
           <ReservationModal manifestation={currentManifestation} />
+          <FindOnShelfModal
+            // TODO: when we have a selected manifestations group, pass it
+            // down here as manifestations prop
+            manifestations={[currentManifestation]}
+            workTitles={work.titles.full}
+            authors={work.creators}
```

Comment:
You can destruct all the work properties in the definition of the Material functional component

Reply:
I'm sorry but I think I prefer it not destructed 🙈